reporters.json

```diff
@@ -380,6 +381,22 @@
             "variations": {}
         }
     ],
+    "Ariz. Adv. Rep.": [
+        {
+            "cite_type": "state",
+            "editions": {
+                "Ariz. Adv. Rep.": {
+                    "end": null,
+                    "start": null
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:az;supreme.court"
+            ],
+            "name": "Arizona Advanced Reports",
```

Comment:
*Advance* Reports, not *Advanced*. I was able to get the start date for this by calling the publisher, Code Co.